requirements.txt

```diff
@@ -49,3 +49,7 @@ PyLTI1p3==1.7.0
 django-webpack-loader==0.6.0
 jsonschema==3.2.0
 django-csp==3.6
+
+# These are for local development
+pylint==2.5.3
+pylint-django==2.3.0
```

Comment:
What about `mypy`?

But more importantly, shouldn't we specify dev dependencies somewhere else? Keeping images for deployment small -- where these packages are mostly irrelevant -- seems like the ideal. I'm not sure what the best practice is these days, though I found this older post: https://stackoverflow.com/questions/17803829/how-to-customize-a-requirements-txt-for-multiple-environments

Most of the options would also require changing `Dockerfile`. We could postpone adding these dependencies to a later date, just change `.vscode`.

Reply:
Did you restart VSCode? Or at least close/re-open the workspace editor. I don't think it re-reads this the first time. 

Yeah, I wasn't sure what to do with these dependencies either. I figured that someone developing on localhost would be pip install'ing the requirements so their IDE doesn't complain about missing dependencies but some aren't needed in prod. 

It's not big but it doesn't look hard to do from that StackOverflow article. I don't think we currently have anything else that's for local dev only and they aren't that big so I didn't think about that.

Reply:
I also think it's possible that VSCode will prompt to install these if it needs them, so this might not be needed either.

Reply:
I created this idea as a new issue. I don't think it's a big deal that these are included as they are very small dependencies, a few MB total. I think if we cared about image size there's a lot more cleanup ideas, including one I issue I just created that cuts down almost 200MB.